elements/pl-graph/pl-graph.mustache

```diff
@@ -1,19 +1,3 @@
-<script>
-$(function() {
-  window.PLGraphRender({
-    uuid: "{{uuid}}",
-    workerURL: "{{workerURL}}",
-    data: {{{ data }}},
-    engine: "{{engine}}",
-  })
-})
-</script>
-
 <div id="pl-graph-{{uuid}}" class="pl-graph">
```

Comment:
I believe the UUID was only necessary because we had client-side JS. Let's remove it?